src/common/transformations/src/transformations/common_optimizations/gelu_fusion.cpp

```diff
@@ -43,12 +43,13 @@ Predicate check_value(float ref, float eps = std::numeric_limits<float>::epsilon
         "has_constant_value(" + std::to_string(ref) + ")");
 }
 
-bool gelu_replacer(ov::pass::pattern::Matcher& m, const std::shared_ptr<ov::Node>& pattern_input_to_relu) {
+bool gelu_replacer(ov::pass::pattern::Matcher& m, const std::shared_ptr<ov::Node>& pattern_input_to_relu,
```

Comment:
Please update `relu` to `gelu`.
```suggestion
bool gelu_replacer(ov::pass::pattern::Matcher& m, const std::shared_ptr<ov::Node>& pattern_input_to_gelu,
```

Reply:
Applied, thanks!